src/nvc.c

```diff
@@ -1098,6 +1100,9 @@ static int coverage(int argc, char **argv)
       case 'd':
          rpt_mask = parse_cover_print_spec(optarg);
          break;
+      case 'l':
+         item_limit = atoi(optarg);
```

Comment:
```suggestion
         item_limit = parse_int(optarg);
```

`parse_int` will give an error message for `--item-limit=zzzzz` and similar.